src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -526,13 +526,19 @@ public async Task<IActionResult> EditPassword(ResetPasswordViewModel model)
             return Forbid();
         }
 
+        // Exclude validating the current password while changing the current user password
```

Comment:
This is a bad practice. I think the current user password should always be required for security when changing some else's password even.

Reply:
It was the current password for the user who want to change his password, but sometimes you need to change others password

Reply:
yes. I suggest we required the current user to provide their password when changing passwords (even for others).

Reply:
Ok